.travis.yml

```diff
@@ -6,7 +6,7 @@ sudo: required
 language: go
 go_import_path: github.com/KohlsTechnology/eunomia
 go:
-- "1.12.x"
+- "1.13.x"
 env:
   global:
   - GO111MODULE=on
```

Comment:
Do we need it with go 1.13?

Reply:
That's an awesome question, thanks for raising it! I'll be looking into this, you may be right. ❤️